src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+Unlike normal damage, LHP can only be dealt/healed to an individual who is alive and not in crit. 
+
+### Revolutionary Codewords & LHP Damage
+
+Revolutionary codewords is the main way a Rev convert crew, and is the primary conversion method the gamemode is balanced around. When becoming a Revolutionary (Headrev or converted) you are provided with a number of codewords that when spoken deals LHP damage to anyone hearing it. These codewords are unique to each Headrev and are passed on to whatever Revolutionary they convert, and any further conversions down the chain get the same codewords passed on. 
+
+Codewords come in three levels; Low, Mid and High Conversion words, with **5** codewords given in each level.
+  - Low Conversion words are meant to be easy to incorporate into normal speech. There should be so many possible low conversion words that can be rolled that it becomes effectively impossible to metagame. Examples: "love", "hello", "great", "hate".
+  - Mid Conversion words deal a decent amount of LHP damage but can be noticed if one pays attention. These would be at the risk of metagaming due to being flavored towards revolutionary speech, so it would be on the rev player to incorporate them well into conversations. Examples: "disrupt", "rise up", "join".
+  - High Conversion words are very clearly revolutionary and deal high LHP damage, *and is able to convert a crewmember in the Convertable state. They should be mostly useful when the revolution is in full swing, but risky for a non-convertable crewmember to overhear. Examples: "viva", "communism", "comrade".
+
+Dealing damage with codeword has a **10** second cooldown, to avoid simply spamming the codewords. Damage is also split among the crew who hear it, making speech directed to a single individual more effective at targeted conversion, while multiple Revolutionaries can shout it out to crowds without the conversion happening to fast. Using multiple codewords in one sentence does not do anything; it simply uses the highest-tier codeword instead.
+
+Codewords are proposed to deal the following damage:
+  - Low: **10** LHP
+  - Mid: **20** LHP
+  - High: **30** LHP, and converts any crew in the Convertable state.
+
+Codewords are able to deal damage even when communicated across different mediums, however the conversion strength is reduced to compensate for how easy it may be to do so:
+
+- A Revolutionary speaking a codeword through a headset radio deals **0.2**x LHP damage. (Note that crew passively heal LHP, as described in the Deconversion section)
+- The first time a Revolutionary uses the Announcement console to send a message with a codeword, it deals **1**x LHP damage, without being divided among the people hearing it. This is a one-time trigger.
```

Comment:
What should the Reporter role do?

Reply:
Specifically spreading PDA messages laiden with disloyal subtext etc seems like a great thing for the reporter to do. Would be good to consider what roles you want to shine in this mode. E.g. the reporter, the bartender?

Reply:
Social roles and departmental roles with a lot of communication have an inherent advantage. I wouldn't mind the news console being able to deal LHP damage, but on the other hand I wouldn't want to discourage players from reading the news any more than they do now...